tests/end-to-end/tests/card.cy.ts

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
should this header be on all files or is it random atm?

Reply:
I will remove that.